<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>selectMenu</class>
 <widget class="QWidget" name="selectMenu">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>154</width>
    <height>521</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_2">
   <item>
    <widget class="QWidget" name="widget" native="true">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QPushButton" name="localBtn">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>本地歌曲</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="onlineBtn">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>在线歌曲</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="collectionBtn">
        <property name="minimumSize">
         <size>
          <width>110</width>
          <height>50</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>110</width>
          <height>50</height>
         </size>
        </property>
        <property name="font">
         <font>
          <pointsize>10</pointsize>
          <weight>75</weight>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>收藏歌曲</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
